scripts/apidocgen/generate.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# This script generates swagger description file and required Go docs files
+# from the coderd API.
+
+set -euo pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+API_MD_TMP_FILE=$(mktemp /tmp/coder-apidocgen.XXXXXX)
+
+cleanup() {
+	rm -f "${API_MD_TMP_FILE}"
+}
+trap cleanup EXIT
+
+echo "Use temporary file: ${API_MD_TMP_FILE}"
+
+(
+	cd "$SCRIPT_DIR/../.."
+
+	go run github.com/swaggo/swag/cmd/swag@v1.8.6 init \
+		--generalInfo="coderd.go" \
+		--dir="./coderd,./codersdk" \
+		--output="./coderd/apidocs" \
+		--outputTypes="go,json" \
+		--parseDependency=true
+) || exit $?
+
+(
+	cd "$SCRIPT_DIR"
+	npm ci
+
+	# Make sure that widdershins is installed correctly.
+	node ./node_modules/widdershins/widdershins.js --version
```

Comment:
Suggestion: NPM "binaries" are typically invoked via `npm exec -- widdershins --version` or `./node_modules/.bin/widdershins --version`, is one available for this package? It might be more robust considering the future.

Reply:
Fixed, TIL!